geometry/identity_body.hpp

```diff
@@ -46,10 +46,12 @@ Trivector<Scalar, ToFrame> Identity<FromFrame, ToFrame>::operator()(
 }
 
 template<typename FromFrame, typename ToFrame>
-template<typename Scalar>
-SymmetricBilinearForm<Scalar, ToFrame> Identity<FromFrame, ToFrame>::operator()(
-    SymmetricBilinearForm<Scalar, FromFrame> const& form) const {
-  return SymmetricBilinearForm<Scalar, ToFrame>(form.coordinates());
+template<typename Scalar, template<typename S, typename F> typename Multivector>
```

Comment:
Funny that it fits on one line here but the declaration has 3 lines.  I vastly prefer the 1-line version.

Reply:
78 < 80.